### DISCUSSION TITLE
우테코 5차 데모데이 회고

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#197] [5. 회고]
쓴지는 좀 된 글인데 이제야 다듬어서 올리네요^^;;
[게시글 링크](https://amaran-th.github.io/%EC%A3%BC%EC%A0%80%EB%A6%AC%EC%A3%BC%EC%A0%80%EB%A6%AC/[Kerdy]%205%EC%B0%A8%20%EB%8D%B0%EB%AA%A8%EB%8D%B0%EC%9D%B4%20%ED%9A%8C%EA%B3%A0/)

**Comment by pnudpwls1103** [comment]
직접 서비스를 운영하면서 해결한 과정을 자세히 적어둔게 너무 유용하네요!! 서비스 더 알려지길 바라며 화이팅 :)
